### ISSUE TITLE
Password Authentication Failed for user 'XNAT'

### ISSUE THREAD

**Issue by shanebenlolo** [NrgXnat/xnat-docker-compose#82]
**Describe the bug**
Hello,
I am following the simple 'Getting Started' Guide for this repo. I am running into an issue where when running `docker compose up -d` using the default configurations, I am met with the following errors on my console upon a successful build:

```
xnat-web_1    | psql: FATAL:  password authentication failed for user "xnat"

xnat-web_1    | Postgres is unavailable - sleeping

xnat-web_1    | psql: FATAL:  password authentication failed for user "xnat"

xnat-web_1    | Postgres is unavailable - sleeping

xnat-db_1     | 2021-09-28 22:08:41.415 UTC [55] FATAL:  password authentication failed for user "xnat"

xnat-db_1     | 2021-09-28 22:08:41.415 UTC [55] DETAIL:  User "xnat" has no password assigned.

xnat-db_1     |  Connection matched pg_hba.conf line 95: "host all all all md5"

```

**To Reproduce**
1. `git clone https://github.com/NrgXnat/xnat-docker-compose`
2. `cd xnat-docker-compose`
3. ` cp default.env .env`
4. `docker-compose up`  <-- leave off the `-d` so you see the feed from your containers
5. you should see the errors I've posted above

**Expected behavior**
upon running `docker-compose up -d` with th edefault config, I should be able to visit `localhost` and see my xnat server page

**Screenshots**
![docker compose server](https://user-images.githubusercontent.com/66500087/135173291-fc2808b7-c128-48d8-9a59-6f3c91626b24.png)

**Docker server environment (please complete the following information):**
 - Host OS: Windows 10
 - Docker version: v20.10.8
 - docker-compose version: 1.29.2, build 5becea4c

**Configuration:**
 - Branch of XNAT docker-compose project: master

Thanks for the help sorry I know this seems like a simple problem

**Comment by scheiblr** [comment]
As long as the pull request isn't accepted, you could just use this fork which solves this issue: https://github.com/scheiblr/xnat-docker-compose

**Comment by rherrick** [comment]
I've been trying to find a time to review these PRs and just haven't had a chance. Hopefully I'll get to it in the next day or so.

**Comment by kelseym** [comment]
Fixed with #84
